src/Examples/Policies/ApiPolicyWithNamedValue.xml

```diff
@@ -0,0 +1,28 @@
+<policies>
+  <inbound>
+    <base />
+    <choose>
+      <when condition="@(!string.IsNullOrWhiteSpace(context.User.Note))">
+        <set-header name="Provider" exists-action="override">
+          <value>{{namevalue3DisplayName}}</value>
```

Comment:
I believe for this example to be correct that it should use the `name` property 'namedvalue3' instead of the `displayName`.

Reply:
I thought the same but surprisingly `displayName` should be passed otherwise it complains that `property {name} does not exist`.

Reply:
![image](https://user-images.githubusercontent.com/2897853/96289765-96e3b480-0fb3-11eb-9c3c-84973041540c.png)

Yep, I stand corrected. Just confirmed in the portal.